pallets/weights/src/pallet_sto.rs

```diff
@@ -60,15 +60,14 @@ impl pallet_sto::WeightInfo for WeightInfo {
     // Storage: Portfolio PortfolioCustodian (r:1 w:0)
     // Storage: Timestamp Now (r:1 w:0)
     // Storage: Sto FundraiserCount (r:1 w:1)
+    // Storage: Asset Tokens (r:1 w:0)
     // Storage: Portfolio PortfolioAssetBalances (r:1 w:0)
     // Storage: Portfolio PortfolioLockedAssets (r:1 w:1)
     // Storage: Sto FundraiserNames (r:0 w:1)
     // Storage: Sto Fundraisers (r:0 w:1)
-    fn create_fundraiser(i: u32) -> Weight {
-        (122_668_000 as Weight)
-            // Standard Error: 1_610_000
-            .saturating_add((4_429_000 as Weight).saturating_mul(i as Weight))
-            .saturating_add(DbWeight::get().reads(10 as Weight))
+    fn create_fundraiser(_i: u32, ) -> Weight {
```

Comment:
The `i` parameter is the number of price tiers.  This shouldn't be ignored.

Reply:
Copied over:
`            .saturating_add((4_429_000 as Weight).saturating_mul(i as Weight))`

Reply:
I am not seeing this change.